### DISCUSSION TITLE
Enable sensors that are disabled by default - Home Assistant

### DISCUSSION THREAD

**Discussion by Anto79-ops** [timmo001/system-bridge#2386] [Q&A] [answered]
Hi,

Reading the [integration information](https://www.home-assistant.io/integrations/system_bridge/) on System Bridge, it says that you can enable certain sensors such as: 

CPU Temperature
CPU Voltage	
GPU Core Clock Speed	
GPU Memory 
.... etc

How can I enable these additional sensors?

thanks

**Comment by Anto79-ops** [comment]
Hi, anyone know how to do this?

thanks

**Comment by bliblabl8** [comment, marked as answer]
Go on the integration, click in the sensors section on the "+11 hidden sensors", then click on the sensor you want to activate, and then on the upper right hand corner on the settings wheel, and there on the button "Activate"

**Comment by Anto79-ops** [comment]
thanks! looks like may have actually done this already....except the temperature sensors is showing up as unknown in HA but can see it in raw values on the web-interface.

**Comment by bliblabl8** [comment]
My temperature sensors also does show unknown. Where can you see the raw values of it? In which web-interface?

**Comment by Anto79-ops** [comment]
in your browser in the same network, try this link with your IP address and api key in any browser

`http://192.168.1.XXX:9170/app/data.html?apiKey=[api key here remove square brakets]&apiPort=9170`

**Reply by bliblabl8** [reply]
Ah cool, thank you! Searched the compete website of system bridge but didn't find this link anywhere. The documentation seems to be incomplete..

The cpu temperature is showing "null" on my proxmox host.
